core/spec.md

```diff
@@ -5127,12 +5137,12 @@ to have any inlinable attributes (except for `RESOURCE*`) that are present in
 the request processed. Absence of the `?nested` query parameter when an
 inlinable attribute is present MUST generate an error.
 
-Note: If the Registry can not return all expected data in one response then it
-MUST generate an HTTP `406 Not Acceptable` error and SHOULD include a error
-message in the HTTP body indicating that the response is too large to be
-sent in one message. In those cases, the client will need to query the
-individual inlinable attributes in isolation so the Registry can leverage
-[pagination](../pagination/spec.md) of the response.
+Note: If the Registry can not return all expected data in one response because
+it is too large then it MUST generate an HTTP `406 Not Acceptable` error and
+SHOULD include a error message in the HTTP body indicating which attributes
```

Comment:
```suggestion
SHOULD include an error message in the HTTP body indicating which attributes
```

Reply:
Fixed the "a" and added some text about ?query